src/http-gateways/trustless-gateway.md

```diff
@@ -39,6 +39,24 @@ The minimal implementation means:
 
 A subset of "HTTP API" of :cite[path-gateway].
 
+## `GET /ipfs/bafkqaaa`
```

Comment:
Move GET/HEAD probing with identity CID to "notes for implementers" section at the end of document. Maybe call it "Inexpensive Health Check for Clients" ?

Reply:
I think a "dedicated probe paths" section would be enough ? See now, as it is RECOMMENDED from the description of GET/HEAD requests.